packages/sacado/src/KokkosExp_View_Fad.hpp

```diff
@@ -484,13 +484,21 @@ typename std::enable_if<
     Kokkos::Impl::ViewSpecializeSacadoFad >::value ||
   std::is_same< typename ViewTraits<T,P...>::specialize ,
     Kokkos::Impl::ViewSpecializeSacadoFadContiguous >::value,
+#if KOKKOS_VERSION >= 40499
+  typename Impl::MirrorViewType<Space,T,P ...>::dest_view_type>::type
+#else
```

Comment:
@pzehner correct me if I'm wrong but my understanding is that you don't need the `#else` branch